attributes/default.rb

```diff
@@ -8,7 +8,6 @@
 
 default['hops']['hdfs']['user']                = node['install']['user'].empty? ? "hdfs" : node['install']['user']
 default['hops']['group']                       = node['install']['user'].empty? ? "hadoop" : node['install']['user']
-default['hops']['group_id']                    = 1234
```

Comment:
Check review in `conda-chef` - basically move the attribute from `metadata.rb`

Reply:
Done